src/services/blocks/BlocksService.spec.ts

```diff
@@ -358,4 +360,41 @@ describe('BlocksService', () => {
 			).toEqual(true);
 		});
 	});
+
+	describe('fetchExrinsicsByIndex', () => {
+		// fetchBlock options
+		const options = {
+			eventDocs: false,
+			extrinsicDocs: false,
+			checkFinalized: false,
+			queryFinalizedHead: false,
+			omitFinalizedTag: false,
+		};
+
+		it('Returns the correct extrinisics object for block 789629', async () => {
+			const block = await blocksService.fetchBlock(
+				blockHash789629,
+				options
+			);
+
+			const extrinsic = blocksService['fetchExtrinsicsByIndex'](block, 0);
```

Comment:
Would it be better to fetch a non-zero index to prove that we can fetch xts out of order?

Reply:
Yea do you think adding another test case would suite useful here, or just changing to a non-zero index?

Reply:
After a quick thought about this one, I will just change the index for this instead of adding an extra test that in essence will do the same exact thing. I like the insight though, and Ill make sure to comment that in the test itself.